pkg/commands/compute/build.go

```diff
@@ -141,36 +141,16 @@ func (c *BuildCommand) Exec(in io.Reader, out io.Writer) (err error) {
 
 	dest := filepath.Join("pkg", fmt.Sprintf("%s.tar.gz", packageName))
 
+	// NOTE: The minimum package requirement is `fastly.toml` and `main.wasm`.
 	files := []string{
 		manifest.Filename,
+		"bin/main.wasm",
```

Comment:
This is the default location that @fastly/js-compute will write to but it can be configured to write to any location. Should we provide a configuration option in the `fastly compute build` command to specify the wasm location?

Reply:
We currently document the expectation that the main.wasm needs to exist inside the `bin` directory:

<img width="1036" alt="Screenshot 2023-02-21 at 14 50 12" src="https://user-images.githubusercontent.com/180050/220377899-6b122bd2-6e26-4ade-8706-d2ae6920340f.png">

So although it might be possible to change this, for this PR I think we're OK 👍🏻